ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -278,6 +280,7 @@ measurement:
   power: "Rate of energy consumption (assumed context is electrical and real, unless otherwise modified)."
   powerflux: "Energy emitted, reflected, transmitted or received, per unit time."
   pressure: "Measured force per unit area within a system or process."
+  pulserate: "Measured for pulses or breathes per minute,"
```

Comment:
breathes? not entirely sure what is meant by that.

Reply:
beats?

Reply:
Done the necessary contextual changes as required in in this use case.